src/main/java/frc/robot/commands/turret/AutoAlignTurret.java

```diff
@@ -27,8 +27,6 @@ public AutoAlignTurret(Turret turret, Limelight limelight) {
         
         () -> {
           double angle = (turret.getAngle() + limelight.getAngleOffset());
-          angle = Math.max(Constants.TURRET_ROTATION_MIN, angle);
-          angle = Math.min(Constants.TURRET_ROTATION_MAX, angle);
 
           return angle;
```

Comment:
Can you further simplify this to just `() -> turret.getAngle() + limelight.getAngleOffset()`?

Reply:
Fixed in commit 80861ac